Fixtures/CLangModules/CLibraryFlat/Foo.c

```diff
@@ -0,0 +1,6 @@
+int foo() {
```

Comment:
Actually I wanted to name it CModules but CModules refers to system libraries having a modulemaps as a wrapper. so I changed it to C Lang which spelled CLang lol, but anyway Clang would make perfect sense here too. Changing.